flask_praetorian/utilities.py

```diff
@@ -105,3 +112,76 @@ def current_custom_claims():
     """
     jwt_data = get_jwt_data_from_app_context()
     return {k: v for (k, v) in jwt_data.items() if k not in RESERVED_CLAIMS}
+
+
+def deprecated(reason):
```

Comment:
This is a really nice utility! I think you could extract it as it's own python package, but I'm glad to have it in here

Reply:
It is from one ... added attribution (had it in the branch on my server which i didn't push up, then switched to macbook and didn't notice), didn't see a need to require yet another package

Reply:
submitted